### PR TITLE
Add wordpress admin theme

### PR DESCRIPTION
**Pull request by KevinBatdorf**
Let me know what you think. WordPress doesn't have modals so I basically just left it as the dark theme on modals and focused on the toast messages. if they ever add modals I'll come back and fix it :) 

I tried to make it look similar whether using the built-in Sweetalert2 icons or the built-in WordPress icons.

One thing I noticed that might be good to add is a class to the container that tells you which icon is active. Then for example, the timer bar color could reflect the current state. Not important though for this.

Some examples using the WP dashicon set against the WordPress admin panel:
Screen Shot 2019-12-09 at 11 38 35 PM
Screen Shot 2019-12-09 at 11 39 57 PM
Screen Shot 2019-12-09 at 11 41 20 PM
Screen Shot 2019-12-09 at 11 39 30 PM

Modified sweetalert icon (border removed):
Screen Shot 2019-12-09 at 11 42 48 PM

Let me know if you want anything tweaked, updated, etc.
 
Closes https://github.com/sweetalert2/sweetalert2-themes/issues/39

### CODE REVIEW

**Comment by limonte** [comment]
Screenshots look sexy! :heart_eyes: Outstanding work @KevinBatdorf THANK YOU!

> One thing I noticed that might be good to add is a class to the container that tells you which icon is active. Then for example, the timer bar color could reflect the current state. Not important though for this.

Sounds like a valid feature-request. Reported: https://github.com/sweetalert2/sweetalert2/issues/1842
I'll implement it right away, it's trivial to add.

Would you mind resolving the conflict in `index.html` after merging #46?

**Comment by KevinBatdorf** [comment]
@limonte I resolved the conflict. I also added a check during the watch function that will make sure the theme folder is available. I kept typing "wordpress" instead of "wordpress-admin" so this fixed that =/

I also updated the packages here and saw the icon fix you added, thanks!

Screen Shot 2019-12-10 at 8 35 02 AM

**Comment by KevinBatdorf** [comment]
You're probably offline at the moment, but if you could hold off on merging this for now I want to update the modals a bit to differentiate it more from the dark theme

**Comment by KevinBatdorf** [comment]
Okay I added some modal styles to make it more of a unique theme. Ready to go when you are! Let me know if you want anything changed.

Here's some screenshots using SweetAlert2 defaults. 

Screen Shot 2019-12-10 at 10 31 51 AM
Screen Shot 2019-12-10 at 10 31 36 AM
Screen Shot 2019-12-10 at 10 31 29 AM
Screen Shot 2019-12-10 at 10 31 21 AM


And some that I tested in my WP application:

Screen Shot 2019-12-10 at 10 42 43 AM
Screen Shot 2019-12-10 at 10 42 53 AM
Screen Shot 2019-12-10 at 10 43 10 AM

**Comment by limonte** [comment]
OUTSTANDING WORK!

Totally love these toasts! :heart_eyes: :heart_eyes: :heart_eyes: 

I'll review your code during this day!

**Comment by KevinBatdorf** [comment]
Thanks @limonte . Looking at it again now I might need to make a couple more adjustments, so I might be pushing a few more minor commits

**Comment by KevinBatdorf** [comment]
Okay maybe you can take a look and see what needs to be improved on. The difficulty seems to be in resizing the icons. I used a scale transform as a shortcut as adjusting the width of the icon directly would require fine-tuning each of the the individual pieces as well. Unless I'm overlooking something.

I'll try to test it in other browsers tomorrow too. Signing off shortly. Thanks!

File: wordpress-admin/wordpress-admin.scss [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+// TIMER POGRESS BAR
+$swal2-timer-progress-bar-height: .25em;
+$swal2-timer-progress-bar-background: $wordpress-blue;
+
+@import '~sweetalert2/src/sweetalert2';
+
+// Overrides
+.swal2-container {
+  padding: 24px 16px;
```
Comment (limonte) [inline wordpress-admin/wordpress-admin.scss]: use `$swal2-container-padding`

Comment (limonte) [inline wordpress-admin/wordpress-admin.scss]: use `$swal2-container-padding`

File: wordpress-admin/wordpress-admin.scss [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
```
Comment (limonte) [inline wordpress-admin/wordpress-admin.scss]: we prefer not setting `font-family` for popups, as they should be aligned with an app, e.g. inherit `font-family`. I think the app `font-family` for Wordpress Admin is `-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;` If so, this line isn't needed.

File: wordpress-admin/wordpress-admin.scss [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+
+.swal2-header {
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  margin-bottom: 8px;
+}
+
+.swal2-title {
+  margin: 0;
```
Comment (limonte) [inline wordpress-admin/wordpress-admin.scss]: use `$swal2-title-margin`

File: wordpress-admin/wordpress-admin.scss [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+.swal2-toast .swal2-title {
+  font-weight: 400 !important;
+  line-height: 13px;
+}
+
+.swal2-icon {
+  height: 32px;
+  transform: scale(0.7) translateY(24px);
+  animation: none !important;
```
Comment (limonte) [inline wordpress-admin/wordpress-admin.scss]: use `$swal2-icon-animations`

File: wordpress-admin/package.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  "version": "3.0.7",
+  "repository": "sweetalert2/sweetalert2-themes",
+  "homepage": "https://sweetalert2.github.io/",
+  "description": "Wordpress admin theme for SweetAlert2",
+  "author": "",
+  "files": [
+    "*.css",
+    "*.scss"
+  ],
+  "main": "wordpress.css",
```
Comment (limonte) [inline wordpress-admin/package.json]: `wordpress-admin.css`

File: wordpress-admin/package.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@sweetalert2/theme-wordpress-admin",
+  "version": "3.0.7",
+  "repository": "sweetalert2/sweetalert2-themes",
+  "homepage": "https://sweetalert2.github.io/",
+  "description": "Wordpress admin theme for SweetAlert2",
```
Comment (limonte) [inline wordpress-admin/package.json]: `Wordpress` -> `WordPress`

File: wordpress-admin/README.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+@import '~@sweetalert2/theme-wordpress-admin/wordpress-admin.scss';
+```
+
+Integration
+------------
+
+To more closely match the WordPress style toast messages, use the following to get started:
+
+```js
+var notify = Swal.mixin({
```
Comment (limonte) [inline wordpress-admin/README.md]: `var` -> `const`

File: package.json [hunk 8]
```diff
@@ -5,24 +5,25 @@
...
+    "@sweetalert2/execute": "^1.0.1",
+    "@sweetalert2/stylelint-config": "^1.1.23",
+    "browser-sync": "^2.26.7",
+    "fancy-log": "^1.3.3",
+    "gulp": "^4.0.2",
+    "gulp-autoprefixer": "^7.0.1",
+    "gulp-clean-css": "^4.2.0",
     "gulp-header": "^2.0.9",
-    "gulp-rename": "^1.2.2",
+    "gulp-rename": "^2.0.0",
```
Comment (limonte) [inline package.json]: reverse this line please, there's something wrong with `gulp-rename@2.0.0`

File: package.json [hunk 9]
```diff
@@ -5,24 +5,25 @@
   "homepage": "https://sweetalert2.github.io/",
```
Comment (limonte) [inline package.json]: you made changes in `package.json` please bump `yarn.lock` as well (`yarn install`), thanks!

File: wordpress-admin/wordpress-admin.scss (hunk 2 above)
Comment (KevinBatdorf) [reply wordpress-admin/wordpress-admin.scss]: @limonte That's right that SweetAlert will inherit the font family while in the WP dashboard but if someone wants the WP style toasts in a non-WP application then it wont look like WordPress anymore unless they set the font. 

For example,

Screen Shot 2019-12-10 at 11 44 18 PM

It looks fine to me, but it's no longer WordPress style. 

I'll remove it and add a note in the readme, but let me know if you want something else.

**Comment by KevinBatdorf** [comment]
Thanks @limonte Let me know if there's anything else.

**Pull request merged**